### PR TITLE
Updated BMLacq776

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
Because Marrassini gave a very detailed description of the contents of this manuscript, I wanted to be precise as well. Unfortunately, I was not able to identify the works or individual textparts of LIT1295 in some cases: 

ms_i1.4.2
ms_i1.4.3 (described as "Michael sent to Samson" by Marrassini)
ms_1.5.2
ms_1.11.2
ms_1.12.2

It is likely that in fact these parts can be found in other witnesses of Homily of Mikāʾel, but I could not check in manuscripts but relied on the quoted incipits for BAVet82 and EMIP00016, because I had no access to images for these two or for other related manuscripts. 

I do not understand which manuscript was indicated by Marrassini with A 84 = N 137 in ms.i1.4.1. It is not mentioned in the introductory text. If this note is not very important, it can also deleted. Also I am not sure, whether I encoded the references to manuscripts correctly. 

In describing the quire structure, I differ from Marrassini in quire 10. His account does not match the total number of folios. 

Even more important is another difference in the description of hands. Marrassini thought, there are at least five hands visible, but according to my opinion, there are only two. For the larger part of the manuscript, the size of the pen changes frequently as well as the saturation of the ink, but special features of hand 1 remains constant throughout the codex. It would be nice, if somebody else may have a look to it and may confirm or falsify my judgement! 

Marrassini also gave a lot of general information on other manuscripts, that I did not record. 

A number of new IDs for works and for persons, related to this manuscript will follow in a minute.

### CODE REVIEW

File: FlorenceBML/BMLacq776.xml [hunk 1]
```diff
@@ -30,10 +31,731 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                                        ሊቅየ፡ ወሊቆሙ፡ ለመላእክት፡ ተሣሃለኒ፡ እስመ፡ ሐካይ፡ ብእሲሁ፡ ወኢይወጽእ፡ እምነ፡ ቤቱ፡ ወአልቦቱ፡ ተግባር፡ ወኢስራኅ፡ ወኢየአምር፡ ንጊደ፡ ወእምዝ፡ ወእምዝ፡ 
+                                        መጽአ፡ ቅዱስ፡ <hi rend="rubric">ሚካኤል፡</hi> ወአስተርአዮ፡ በኅልም፡ ወይቤሎ፡ አንተ፡ ብእሲ፡ ሀካይ፡ ወኢትትገበር፡ አሐደ፡ እምግብርናት፡ ወትብለኒ፡ 
+                                        መአልተ፡ ወሌሊተ፡ ተሣሃለኒ፡ አይነ፡ </incipit>
+                                    <explicit xml:lang="gez">ወወሀቦ፡ ለቤተ፡ ክርስቲያን፡ ወአግበረ፡ ፃሕለ፡ ዘቅዱስ፡ <hi rend="rubric">ሚካኤል፡</hi> ወነዳይኒ፡ ነሥአ፡ ሃምሳ፡ 
+                                        ረ<pb n="42r"/>በሁ፡ አግበረ፡ ጽዋዓ፡ ለቤተ፡ ክርስቲያን፡ ዘቅዱስ፡ <hi rend="rubric">ሚኤል፡</hi> ወተዓረኩ፡ ፪ሆሙ፡ አምዓሜሃ፡ እለት፡ ዕሰሙ፡ ወፍቅሩ፡ 
+                                        የሀሉ፡ ምስለ፡ ፍቁሩ፡ <gap reason="omitted" unit="chars" quantity="4"/> ለአለመ፡ ዓሚ፡ ከመ፡ </explicit>
+                                    <note>Anonymous here, but elsewhere as in <ref target="#">A 84 = N 137</ref> attributed to "Severus, patriarch of Alexandria"
+                                        or to a "Bishop of Aksum" as in <ref target="#EMML1433"></ref>.</note>
```
Comment (eu-genia) [inline FlorenceBML/BMLacq776.xml:195]: `ref type="mss" corresp="EMML1433"` (`corresp`, no `#`)

File: FlorenceBML/BMLacq776.xml [hunk 2]
```diff
@@ -30,10 +31,731 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                                        ዓሜን፡ ወኮነ፡ ፩ዱ፡ ነዳይ፡ ውስተ፡ ፩ብሔር፡ ወዓቢይ፡ ሃይማኖቱ፡ ይጼሊ፡ መአልተ፡ ወሌሊተ፡ ወይብል፡ ኦ<gap reason="omitted" unit="chars" quantity="4"/> 
+                                        ሊቅየ፡ ወሊቆሙ፡ ለመላእክት፡ ተሣሃለኒ፡ እስመ፡ ሐካይ፡ ብእሲሁ፡ ወኢይወጽእ፡ እምነ፡ ቤቱ፡ ወአልቦቱ፡ ተግባር፡ ወኢስራኅ፡ ወኢየአምር፡ ንጊደ፡ ወእምዝ፡ ወእምዝ፡ 
+                                        መጽአ፡ ቅዱስ፡ <hi rend="rubric">ሚካኤል፡</hi> ወአስተርአዮ፡ በኅልም፡ ወይቤሎ፡ አንተ፡ ብእሲ፡ ሀካይ፡ ወኢትትገበር፡ አሐደ፡ እምግብርናት፡ ወትብለኒ፡ 
+                                        መአልተ፡ ወሌሊተ፡ ተሣሃለኒ፡ አይነ፡ </incipit>
+                                    <explicit xml:lang="gez">ወወሀቦ፡ ለቤተ፡ ክርስቲያን፡ ወአግበረ፡ ፃሕለ፡ ዘቅዱስ፡ <hi rend="rubric">ሚካኤል፡</hi> ወነዳይኒ፡ ነሥአ፡ ሃምሳ፡ 
+                                        ረ<pb n="42r"/>በሁ፡ አግበረ፡ ጽዋዓ፡ ለቤተ፡ ክርስቲያን፡ ዘቅዱስ፡ <hi rend="rubric">ሚኤል፡</hi> ወተዓረኩ፡ ፪ሆሙ፡ አምዓሜሃ፡ እለት፡ ዕሰሙ፡ ወፍቅሩ፡ 
+                                        የሀሉ፡ ምስለ፡ ፍቁሩ፡ <gap reason="omitted" unit="chars" quantity="4"/> ለአለመ፡ ዓሚ፡ ከመ፡ </explicit>
+                                    <note>Anonymous here, but elsewhere as in <ref target="#">A 84 = N 137</ref> attributed to "Severus, patriarch of Alexandria"
```
Comment (eu-genia) [inline FlorenceBML/BMLacq776.xml:194]: As is clear from the list of abbreviations A stands for d'Abbadie and N for Conti Rossini Notice, so the reference is to BNFabb84 https://archivesetmanuscrits.bnf.fr/ark:/12148/cc103059f

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: FlorenceBML/BMLacq776.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply FlorenceBML/BMLacq776.xml:195]: Done.

File: FlorenceBML/BMLacq776.xml [hunk 3]
```diff
@@ -30,10 +31,731 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                                            እጓለ፡ እመሕያው፡ ዝይቅውምም፡ ኩሎ፡ ጊዜ፡ ቅድመ፡ መንበረ፡ እባዩ፡ ለአብ፡ ይትነበል፡ በእንተ፡ ኩሉ፡ ፍጥረት፡ ዝንቱ፡ ዘአርአዩ፡ ለኢያሱ፡ </incipit>
+                                        <explicit xml:lang="gez">እስከ፡ ይፌጽሙ፡ ገድሎሙ፡ ወተገብረ፡ ሎቱ፡ ተዝካረ፡ ባወምጽዋታት፡ በእንተ፡ ስሙ፡ በበ፲ወ፪ቱ፡ ወርኅ፡ እስመ፡ ውእቱ፡ ይስእሎ፡ 
+                                            ለእ<hi rend="ligature">ግዚ</hi>አብሔር፡ በእንተ፡ ፍሬያተ፡ ምድር፡ ወበእንተ፡ ማያተ፡ አፍላግ፡ ወበእንተ፡ ነፍሳት፡ ሠናያን፡ ከመ፡ ይፈጽሞሙ፡ 
+                                            እ<hi rend="ligature">ግዚ</hi>አብሔር፡ ለኩሎሙ፡ በርትሰ፡ ትንብልናሁ፡ ያሃሉ፡ ምስ፡ ገብ<del rend="strikethrough">ሩ</del>ሩ፡ 
+                                            <gap reason="omitted"/> ለዓለመ፡ ዓሜን፡ </explicit>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.1.4">
+                                        <locus from="12rb" to="13rb"></locus>
+                                        <title ref="LIT1295Dersan#MiracleHedar"></title>
+                                        <note>Attested also in <ref target="#EMIP00016"/> on f. 9v to 13r.</note>
```
Comment (thea-m) [inline FlorenceBML/BMLacq776.xml:79]: `<ref type="mss" corresp="EMIP00016"/>` would be correct here. `<ref target="#"/>` is only used to refer to xml:ids within the same file. I know it's confusing, it comes with time

File: FlorenceBML/BMLacq776.xml [hunk 4]
```diff
@@ -30,10 +31,731 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                                        <hi rend="rubric">በወርኃ፡ ጥርቦሶላመ፡  እ<hi rend="ligature">ግዚ</hi>አብሔር፡ </hi>ዓሜን፡ መፍትው፡ እንከ፡ ትሰምእዎ፡ በፍርሐት፡ 
+                                        ከመ፡ ታአምሩ፡ አበዮ፡ ለቅዱስ፡ <hi rend="rubric">ሚካኤል፡ ሊቀ፡ መላ</hi>እክት፡ እም፡ አብያኢሁ፡ ሊቃነ፡ መላእክት፡ </incipit>
+                                    <explicit xml:lang="gez">ኦሪትኒ፡ ትቤ፡ ተፈጸመ፡ ኩሉ፡ በእንተ፡ ስቡዕ፡ ወንህነሂ፡ ተፈርነ፡ በእለተ፡ አርብ፡ ሠእር፡ ወሐመልማል፡ ፀሀይ፡ ወወርህ፡ ወክዋክብት፡ 
+                                        ሶበ፡ ርእየ፡ ሰኖሙ፡ ወብርሃኖሙ፡ ይቤ፡ አነብር፡ መንባርየ፡ ዲቤሆሙ፡ ወእትሜስሎ፡ ለኃይል፡ ወአሜሃ፡ ወድቀ፡ ወበሐሙስ፡ ፈወረ፡ ኩሉ፡ ዘይትሐወሰ፡ ምስለ፡ ሲሳዮሙ፡ 
+                                        ወበአርብ፡ ፈጠረ፡ አዳምሃ፡ ምስለ፡ ኵለ፡ ዘሠርር፡ ወበሰኅቦት፡ አእረፈ፡ እምኩሉ፡ ግብሩ፡ ዘሎቱ፡ ሰብሐት፡ ወክብር፡ ለዓለመ፡ ዓለም፡ ዓማን፡ </explicit>
+                                    <note>Homily on the angels and the child <persName ref="PRS11321Talafinos"/>, who was found at the sea. 
+                                        Here anonymous, but elsewhere attributed to <persName ref="PRS3828Epiphani"/>. The story is a variant of 
+                                        <title ref="LIT4155MiraclesMichael"/> as found in <locus target="#ms_i1.5.1"/>;
```
Comment (thea-m) [inline FlorenceBML/BMLacq776.xml:162]: here ref instead of locus

File: FlorenceBML/BMLacq776.xml [hunk 5]
```diff
@@ -30,10 +31,731 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                                        ከመ፡ ታአምሩ፡ አበዮ፡ ለቅዱስ፡ <hi rend="rubric">ሚካኤል፡ ሊቀ፡ መላ</hi>እክት፡ እም፡ አብያኢሁ፡ ሊቃነ፡ መላእክት፡ </incipit>
+                                    <explicit xml:lang="gez">ኦሪትኒ፡ ትቤ፡ ተፈጸመ፡ ኩሉ፡ በእንተ፡ ስቡዕ፡ ወንህነሂ፡ ተፈርነ፡ በእለተ፡ አርብ፡ ሠእር፡ ወሐመልማል፡ ፀሀይ፡ ወወርህ፡ ወክዋክብት፡ 
+                                        ሶበ፡ ርእየ፡ ሰኖሙ፡ ወብርሃኖሙ፡ ይቤ፡ አነብር፡ መንባርየ፡ ዲቤሆሙ፡ ወእትሜስሎ፡ ለኃይል፡ ወአሜሃ፡ ወድቀ፡ ወበሐሙስ፡ ፈወረ፡ ኩሉ፡ ዘይትሐወሰ፡ ምስለ፡ ሲሳዮሙ፡ 
+                                        ወበአርብ፡ ፈጠረ፡ አዳምሃ፡ ምስለ፡ ኵለ፡ ዘሠርር፡ ወበሰኅቦት፡ አእረፈ፡ እምኩሉ፡ ግብሩ፡ ዘሎቱ፡ ሰብሐት፡ ወክብር፡ ለዓለመ፡ ዓለም፡ ዓማን፡ </explicit>
+                                    <note>Homily on the angels and the child <persName ref="PRS11321Talafinos"/>, who was found at the sea. 
+                                        Here anonymous, but elsewhere attributed to <persName ref="PRS3828Epiphani"/>. The story is a variant of 
+                                        <title ref="LIT4155MiraclesMichael"/> as found in <locus target="#ms_i1.5.1"/>; 
+                                        <title ref="LITLIT1295Dersan#CommemorativeSane"/> as found in <locus target="#ms_i1.8.2"/> and
```
Comment (thea-m) [inline FlorenceBML/BMLacq776.xml:163]: here, `<ref type="work" corresp="LIT1295Dersan#CommemorativeSane"/>`

File: FlorenceBML/BMLacq776.xml [hunk 6]
```diff
@@ -30,10 +31,731 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                                <msItem xml:id="ms_i1.12.3">
+                                    <locus from="106ra" to="107ra"/>
+                                    <title ref="LIT1295Dersan#CommemorativeTeqemt"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.12.4">
+                                    <locus target="#107r"/>
+                                    <title ref="LIT1295Dersan#SalamTeqemt"/>
+                                </msItem>
+                            </msItem>
+                            <note>Spaces left free for owner names throughout except for <locus target="#52rb #105va #106ra"/>, where the name of the owner
```
Comment (thea-m) [inline FlorenceBML/BMLacq776.xml:571]: This information can to an "extra"

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you, wonderful! Just some technical remarks. (I did not check the Ethiopic texts for typos)

File: FlorenceBML/BMLacq776.xml (hunk 3 above)
Comment (CarstenHoffmannMarburg) [reply FlorenceBML/BMLacq776.xml:79]: Done!

Comment (CarstenHoffmannMarburg) [reply FlorenceBML/BMLacq776.xml:79]: also in line 458.

File: FlorenceBML/BMLacq776.xml (hunk 4 above)
Comment (CarstenHoffmannMarburg) [reply FlorenceBML/BMLacq776.xml:162]: I made this change 12 times in this record. I hope, I did not miss any!

File: FlorenceBML/BMLacq776.xml (hunk 5 above)
Comment (CarstenHoffmannMarburg) [reply FlorenceBML/BMLacq776.xml:163]: Also corrected this mistake 12x.

File: FlorenceBML/BMLacq776.xml (hunk 6 above)
Comment (CarstenHoffmannMarburg) [reply FlorenceBML/BMLacq776.xml:571]: I encoded it as an ownership note even though these are non existing ownership notes.

**Review by thea-m — APPROVED** [review]

**Review by eu-genia — APPROVED** [review]

**Pull request merged**